DragaliaAPI/DragaliaAPI.Shared/Resources/Story/QuestStoryRewardInfo.json

```diff
@@ -1,617 +1,727 @@
- },
- {
-  "_Id": 2084301,
-  "_Rewards": [
-   {
-    "_Type": 9,
-    "_Id": 102701,
-    "_Quantity": 1
-   }
-  ]
- },
- {
-  "_Id": 2084401,
-  "_Rewards": [
-   {
-    "_Type": 9,
-    "_Id": 102801,
-    "_Quantity": 1
-   }
-  ]
- },
- {
-  "_Id": 2084501,
-  "_Rewards": [
-   {
-    "_Type": 9,
-    "_Id": 102901,
-    "_Quantity": 1
-   }
-  ]
- },
- {
-  "_Id": 2084601,
-  "_Rewards": [
-   {
-    "_Type": 9,
-    "_Id": 103001,
-    "_Quantity": 1
-   }
-  ]
- }
-]
\ No newline at end of file
+  {
+    "_Id": 1000103,
+    "_Rewards": [
+      {
+        "_Type": 1,
+        "_Id": 10540201,
+        "_Quantity": 1
+      }
+    ]
+  },
+  {
+    "_Id": 1000106,
+    "_Rewards": [
+      {
+        "_Type": 1,
+        "_Id": 10440301,
+        "_Quantity": 1
+      }
+    ]
+  },
+  {
+    "_Id": 1000111,
+    "_Rewards": [
+      {
+        "_Type": 1,
+        "_Id": 10840501,
+        "_Quantity": 1
+      }
+    ]
+  },
+  {
+    "_Id": 1000202,
+    "_Rewards": [
+      {
+        "_Type": 1,
+        "_Id": 10640401,
+        "_Quantity": 1
+      }
+    ]
+  },
+  {
+    "_Id": 1000808,
+    "_Rewards": [
+      {
+        "_Type": 1,
+        "_Id": 10340502,
+        "_Quantity": 1
+      },
+      {
+        "_Type": 9,
+        "_Id": 100601,
+        "_Quantity": 1
+      }
+    ]
+  },
+  {
+    "_Id": 1001108,
+    "_Rewards": [
+      {
+        "_Type": 1,
+        "_Id": 10240101,
+        "_Quantity": 1
+      }
+    ]
+  },
+  {
+    "_Id": 1001410,
+    "_Rewards": [
+      {
+        "_Type": 1,
+        "_Id": 10750504,
+        "_Quantity": 1
+      }
+    ]
+  },
+  {
+    "_Id": 1001610,
+    "_Rewards": [
+      {
+        "_Type": 1,
+        "_Id": 10650503,
+        "_Quantity": 1
+      }
+    ]
+  },
+  {
+    "_Id": 1000109,
+    "_Rewards": [
+      {
+        "_Type": 7,
+        "_Id": 20040301,
+        "_Quantity": 1
+      }
+    ]
+  },
+  {
+    "_Id": 1000210,
+    "_Rewards": [
+      {
+        "_Type": 7,
+        "_Id": 20040201,
+        "_Quantity": 1
+      }
+    ]
+  },
+  {
+    "_Id": 1000311,
+    "_Rewards": [
+      {
+        "_Type": 7,
+        "_Id": 20040101,
+        "_Quantity": 1
+      }
+    ]
+  },
+  {
+    "_Id": 1000412,
+    "_Rewards": [
+      {
+        "_Type": 7,
+        "_Id": 20040401,
+        "_Quantity": 1
+      }
+    ]
+  },
+  {
+    "_Id": 1000509,
+    "_Rewards": [
+      {
+        "_Type": 7,
+        "_Id": 20040501,
+        "_Quantity": 1
+      }
+    ]
+  },
+  {
+    "_Id": 1000607,
+    "_Rewards": [
+      {
+        "_Type": 9,
+        "_Id": 100603,
+        "_Quantity": 1
+      }
+    ]
+  },
+  {
+    "_Id": 1000709,
+    "_Rewards": [
+      {
+        "_Type": 9,
+        "_Id": 100602,
+        "_Quantity": 1
+      }
+    ]
+  },
+  {
+    "_Id": 1000909,
+    "_Rewards": [
+      {
+        "_Type": 9,
+        "_Id": 100604,
+        "_Quantity": 1
+      }
+    ]
+  },
+  {
+    "_Id": 1001009,
+    "_Rewards": [
+      {
```

Comment:
Did you add these in manually? Where did you get the info from?

Reply:
Yes, I added the rewards manually. I looked at FortPlantDetail.json to determine the material IDs and count for each level up, dropping the info in a spreadsheet to get the totals.

Is it ok to add to that file?

Reply:
Sure, I think as it has an *Info suffix it was manually generated anyway.